env/HERA.env

```diff
@@ -102,6 +102,13 @@ elif [[ "${step}" = "atmanlfv3inc" ]]; then
     [[ ${NTHREADS_ATMANLFV3INC} -gt ${nth_max} ]] && export NTHREADS_ATMANLFV3INC=${nth_max}
     export APRUN_ATMANLFV3INC="${launcher} -n ${npe_atmanlfv3inc} --cpus-per-task=${NTHREADS_ATMANLFV3INC}"
 
+elif [[ "${step}" = "prepaeroobs" ]]; then
```

Comment:
@aerorahul I forget, did you say you wanted all these to move towards `prepXobs` or `prepobsX`?

Reply:
`prepobsX` or `prepobs_X`
having a `delimiter` makes easy for cutting if needed down the road.

Reply:
Also easier to read.

Reply:
Thanks, I will update the step name.